packages/runtime/datastore/src/channelContext.ts

```diff
@@ -31,6 +31,8 @@ export interface IChannelContext {
     reSubmit(content: any, localOpMetadata: unknown): void;
 
     getGCData(): Promise<IGarbageCollectionData>;
+
+    updateUsedRoutes(usedRoutes: string[]): void;
```

Comment:
Can you please put comment above it describing
1. What is expected from implementers
2. What is expected from callers, including when this API gets called.